app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import PropTypes from "prop-types"
+import {useSelector} from 'react-redux';
+const Greeting =(props)=> {
+    const message = useSelector((state) => state.messageReducer.message);
+    console.log("message::",message);
+    return (
+      <div>
+        Message: {message.text}
+      </div>
+    );
+}
+
+/* Greeting.propTypes = {
+  message: PropTypes.string
+}; */
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.